src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -28,11 +28,11 @@ public class WebFunctionsManager : IWebFunctionsManager
         private readonly HostNameProvider _hostNameProvider;
         private readonly IFunctionMetadataManager _functionMetadataManager;
 
-        public WebFunctionsManager(IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, ILoggerFactory loggerFactory, HttpClient client, ISecretManagerProvider secretManagerProvider, IFunctionsSyncManager functionsSyncManager, HostNameProvider hostNameProvider, IFunctionMetadataManager functionMetadataManager)
+        public WebFunctionsManager(IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, ILoggerFactory loggerFactory, IHttpClientFactory clientFactory, ISecretManagerProvider secretManagerProvider, IFunctionsSyncManager functionsSyncManager, HostNameProvider hostNameProvider, IFunctionMetadataManager functionMetadataManager)
```

Comment:
nit: recommend you be consistent with your naming. Half of the params are named clientFactory, and half are named httpClientFactory

Reply:
Saw this comment just after merging - following up with a separate PR. Thanks @mathewc !